src/stdlib/RandomMeasures.jl

```diff
@@ -211,42 +201,35 @@ function distribution(d::SizeBiasedSamplingProcess{<:PitmanYorProcess})
     return LocationScale(zero(d_rpm_d), d.surplus, dist)
 end
 
-function _logpdf_table(d::PitmanYorProcess, m::AbstractVector{Int})
+function _logpdf_table(d::PitmanYorProcess{T}, m::AbstractVector{Int}) where {T<:AbstractFloat}
```

Comment:
Same as above, in case you want to add a static type parameter, it should probably be `T<:Real`.